manifests/init.pp

```diff
@@ -149,4 +149,6 @@
     -> Class['marathon::config']
     ~> Class['marathon::service']
     -> anchor { 'marathon::end': }
+
+  Package['marathon'] ~> Class['marathon::service']
```

Comment:
Can we rather do `Class['marathon::install'] ~> Class['marathon::service']`?

I'd like to keep the details of the install class out of this main class.